button.md

```diff
@@ -0,0 +1,85 @@
+## Quick Start
+
```

Comment:
I think both here and widget doc should include the H1 mentioning the name e.g.

# Harvest Button Developer Documentation